docs/eventing/getting-started.md

```diff
@@ -0,0 +1,438 @@
+metadata:
+  name: foo-display
+spec:
+  selector:
+    app: foo-display
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 8080
+END
+```
+
+2. To create the `bar-display` subcomponent, enter the following command:
+
+```sh
+kubectl -n default apply -f - << END
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: bar-display
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      app: bar-display
+  template:
+    metadata:
+      labels: *labels
+    spec:
+      containers:
+        - name: event-display
+          # Source code: https://github.com/knative/eventing-sources/blob/release-0.6/cmd/event_display/main.go
+          image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display@sha256:37ace92b63fc516ad4c8331b6b3b2d84e4ab2d8ba898e387c0b6f68f0e3081c4
+
+---
+
+# Service pointing at the previous Deployment. This will be the target for event
+# consumption.
+kind: Service
+apiVersion: v1
+metadata:
+  name: bar-display
+spec:
+  selector:
+    app: bar-display
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 8080
+END
+```
+
+3. Just like the `Broker`, verify that the event consumers are working with the following command:
+
+```sh
+kubectl -n default get deployments foo-display bar-display
+```
+
+This commmand shows all of the deployments that have been created in `foo-display` and `bar-display`. You should have one replica in each event consumer:
+
+```sh
+NAME           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+foo-display    1         1         1            1           26s
+NAME           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+bar-display    1         1         1            1           16s
+```
+
+The number of replicas in the **DESIRED** column should match the number of replicas in the **AVAILABLE** column. This may take a few minutes. If after two minutes the numbers still do not match the example, then see the [Debugging Guide](./debugging/README.md).
+
+### Creating `Triggers`
+
+`Triggers` allow events to register interest with a `Broker`. A `Trigger` is split into two parts: the `Filter`, which tracks interested events, and the `Subscriber`, which determines where the event should be sent. Triggers can search for various types of events:
+
+1. To create the first `Trigger` enter the following command:
+
+```sh
+kubectl -n default apply -f - << END
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Trigger
+metadata:
+  name: foo-display
+spec:
+  filter:
+    sourceAndType:
+      type: foo
+  subscriber:
+    ref:
+     apiVersion: v1
+     kind: Service
+     name: foo-display
+END
+```
+
+Take notice of the attributes of the `Filter`. Every valid `CloudEvent` has attributes named `Type` and `Source`. Triggers allow you to specify interest in specific `CloudEvents` by matching its `Type` and `Source`. 
+
+In this case, the `Trigger` you created searches for all `CloudEvents` of type `foo` and sends them to the `event consumer` `foo-display`.
```

Comment:
I think you could also call foo-display a "subscriber." We could ask Akash when we should say "event consumer" and when we should say "subscriber". I do like that subscriber is a term that shows up in the .yaml, so I feel like that could be the clearest term for users, but I don't know for sure.

Reply:
@akashrv and I talked about this. Because of concerns with users mistaking "subscriber" in this context and other types of subscribers, it's best to keep it as event consumer globally.